src/miniwiki.rs

```diff
@@ -284,4 +291,14 @@ fn render_arguments(parameters: &Vec<DumpFunctionParameter>) -> Fragment {
                 }
             </div>
         )))
+}
+
+fn member_element_class<'a>(tags: &'a BTreeSet<String>, main_class: &str) -> String {
+    let mut element_class: String = "dump-class-member ".to_owned();

```

Comment:
```suggestion
    let mut element_class = "dump-class-member ".to_owned();
```

Should be able to drop this type annotation